src/tools/cursors/index.js

```diff
@@ -0,0 +1,384 @@
+import MouseCursor from './MouseCursor.js';
+
+export const angleCursor = new MouseCursor(
```

Comment:
A tiny legend of values like `ACTIVE_COLOR` at the top of this file, and I'm loving this.

Reply:
Done.